adstb/bumblebee.py

```diff
@@ -212,8 +231,34 @@ def build_meta_tags(solrdoc):
         
     if 'doi' in data:
         data['doi'] = data['doi'][0]
+    if 'identifier' in data:
+        # expected format: ["2011arXiv1108.0669H", "2012ApJS..199...26H", "10.1088/0067-0049/199/2/26", "arXiv:1108.0669"]
+        for i in data['identifier']:
+            if i.startswith('arXiv'):
+                data['arxiv_id'] = i
+    if 'keyword' in data:
+        data['keyword'] = ', '.join(data['keyword'])
+    if 'page' in data:
+        data['page'] = data['page'][0]
+    if 'page_range' in data and '-' in data['page_range']:
+        # expected format: 41-59
+        data['lastpage'] = data['page_range'].split('-')[1]
+    if 'title' in data:
+        data['title'] = _html_to_text(data['title'][0])
     return ''.join(meta_template(data))
 
+
+htmlParser = HTMLParser()
+tag_re = re.compile(r'<[^>]+>')
+
+
+def _html_to_text(t):
+    """remove html tags and clean html encoded characters and return plain text string"""
+    t = htmlParser.unescape(t)
+    t = tag_re.sub('', t)
```

Comment:
shouldn't the lines 257-258 be swapped?

Reply:
I like what it currently does, converting u'Renormalization of quark propagators from twisted-mass lattice QCD at N&lt;SUB&gt;f&l\
t;/SUB&gt;=2' to u'Renormalization of quark propagators from twisted-mass lattice QCD at Nf=2'.  What did you have in mind?